ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -291,6 +299,31 @@ def _extract_events(events: Sequence[Event]) -> ProtoSpan.TimeEvents:
     )
 
 
+OT_RESOURCE_LABEL_TO_GCP = {
```

Comment:
When you move monitoring and trace into the same package, this can exist in one place.

Reply:
Ah yeah, @aabmass already flagged that, tracked here https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/issues/11